packages/react/src/contexts/CorbadoSessionContext.tsx

```diff
@@ -18,6 +18,14 @@ export interface CorbadoSessionContextProps {
   getPasskeys: (abortController?: AbortController) => Promise<Result<PassKeyList, CorbadoError>>;
   deletePasskey: (id: string) => Promise<Result<void, PasskeyDeleteError>>;
   getFullUser: (abortController?: AbortController) => Promise<Result<CorbadoUser, CorbadoError>>;
+  getIdentifierListConfig: (abortController?: AbortController) => Promise<Result<IdentifierListConfig, CorbadoError>>;
```

Comment:
in the frontend we use Id instead of ID => so let's make it identifierId instead of identifierID please